tests/snapshots/tests__standard@function-call-multiline-comment.lua.snap

```diff
@@ -7,3 +7,25 @@ expression: format(&contents)
 
 call(item, --[[param=]] false)
 
+call(--[[ we don't use it ]] true)
+
+call(
+	--[[
+		this comment spans
+		multiple lines
+	]]
+	false
+)
+
+x(
+	true,
+	90210,
+	--[[
+		color wheel is time-reversed
+	]]
```

Comment:
Interesting that it moves these outside of the comma, but maybe not a big deal?

Reply:
Tbh, I think the likelihood of a comment in this particular position (after expression but before the comma) is very low, so not sure it matters too much.

I moved it afterwards for simplicity, but I made an assumption here - let me see if it still works in its current position